puppet/spec/unit/util/puppetdb/config_spec.rb

```diff
@@ -185,34 +181,34 @@ def write_config(content)
       it "should fail if given an invalid hostname" do
         write_config <<CONF
 [main]
-server_urls = https://invalid_host_name.com
+server_urls = https://invalid_host\[\]_name.com
```

Comment:
Ruby 2's URI library doesn't think `invalid_host_name` is invalid, so my tests were failing.